### PR TITLE
Baseline - AWS > CIS v3.0 > Section 5 enforcements. Closes #708

### LINKED ISSUE #708: Baseline - AWS > CIS v3.0 > Section 5 enforcements
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. 

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Workspace Reference URL**
- [ ] Create README.md (Required)
- [ ] Add `outputs.tf` (If applicable)
- [ ] Add `variables.tf` (Required)
- [ ] Add `providers.tf` (Required)
- [ ] Add `smart_folder.tf` (Required)
- [ ] Policies end with `*_policies.tf` (Required)
- [ ] Cloud specific resources are in own files
- [ ] No customer data present

### PR DESCRIPTION
**Pull request by Subhajit97**

### CODE REVIEW

**Review by SumitPopat — CHANGES_REQUESTED** [review]

**Review by SumitPopat — APPROVED** [review]

File: cis_enforcements/aws/cis_v300_s5_networking/ec2_account_attributes.tf [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# AWS > EC2 > Account Attributes > Instance Metadata Service Defaults
+resource "turbot_policy_setting" "aws_ec2_account_attributes_instance_metadata_service_defaults" {
+  resource = turbot_smart_folder.aws_cis_v300_s5_networking.id
+  type     = "tmod:@turbot/aws-ec2#/policy/types/ec2AccountAttributesInstanceMetadataServiceDefaults"
+  value    = "Check: Enabled for V2 only"
+  # value    = "Enforce: Enabled for V2 only"
+  note     = "AWS CIS v3.0.0 - Controls: 5.6"
```
Comment (dboeke) [inline cis_enforcements/aws/cis_v300_s5_networking/ec2_account_attributes.tf:7]: Note should be above the value.

File: cis_enforcements/aws/cis_v300_s5_networking/network_acl.tf [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# AWS > VPC > Network ACL > Ingress Rules > Approved
+resource "turbot_policy_setting" "aws_vpc_network_acl_ingress_rules_approved" {
+  resource = turbot_smart_folder.aws_cis_v300_s5_networking.id
+  type     = "tmod:@turbot/aws-vpc-security#/policy/types/networkAclIngressRulesApproved"
+  value    = "Check: Approved"
+  # value    = "Enforce: Delete unapproved"
+  note     = "AWS CIS v3.0.0 - Controls: 5.1"
```
Comment (dboeke) [inline cis_enforcements/aws/cis_v300_s5_networking/network_acl.tf:7]: order of the attributes should be:

1. resource
2. type
3. note
4. value
5. template input
6. template

File: cis_enforcements/aws/cis_v300_s5_networking/security_group.tf [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+# AWS > VPC > Security Group > Ingress Rules > Approved > Rules
+resource "turbot_policy_setting" "aws_vpc_security_group_ingress_rules_approved_rules" {
+  resource = turbot_smart_folder.aws_cis_v300_s5_networking.id
+  type     = "tmod:@turbot/aws-vpc-security#/policy/types/securityGroupIngressRulesApprovedRules"
+  note     = "AWS CIS v3.0.0 - Controls: 5.2, 5.3, 5.4"
+  template_input = <<-EOT
+    {
+      value: constant(value: "REJECT *")
```
Comment (dboeke) [inline cis_enforcements/aws/cis_v300_s5_networking/security_group.tf:17]: no reason to do this as a constant.  only if we have separate enforce/check options in the policy setting.  in this case the calc policy will be the same regardless of check/enforce.

File: cis_enforcements/aws/cis_v300_s5_networking/security_group.tf [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+# AWS > VPC > Security Group > Egress Rules > Approved > Rules
+resource "turbot_policy_setting" "aws_vpc_security_group_egress_rules_approved_rules" {
+  resource = turbot_smart_folder.aws_cis_v300_s5_networking.id
+  type     = "tmod:@turbot/aws-vpc-security#/policy/types/securityGroupEgressRulesApprovedRules"
+  note     = "AWS CIS v3.0.0 - Controls: 5.4"
+  template_input = <<-EOT
+    {
+      value: constant(value: "REJECT *")
```
Comment (dboeke) [inline cis_enforcements/aws/cis_v300_s5_networking/security_group.tf:58]: same comment here about no need for this constant value, as it doesn't change.

**Review by dboeke — CHANGES_REQUESTED** [review]

**Review by dboeke — APPROVED** [review]

**Pull request merged**